myhpi/search/templates/search/search_result.html

```diff
@@ -9,7 +9,7 @@ <h5 class="text-break">{{ result.title }}</h5>
                 {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
             </time>
         </hgroup>
```

Comment:
The mobile menu needs its background back ^^

![grafik](https://github.com/user-attachments/assets/0d3464ed-eb01-4478-bb8f-c8c7a28fbf63)